addons/website/views/snippets.xml

```diff
@@ -1191,7 +1185,20 @@
 
     <!-- Facebook Page -->
     <div data-js="facebookPage" data-selector=".o_facebook_page">
-        <we-button data-fb-page-options="true" data-no-preview="true">Facebook Options</we-button>
+        <we-title>Page URL</we-title>
+        <we-input data-page-url="https://www.facebook.com/Odoo" data-no-preview="true"/>
+        <we-title class="o_we_section_title">Tabs</we-title>
+        <we-section class="pl-1 mt-0">
```

Comment:
Not sure about the style of this, the indentation make the options appear broken to me (but maybe that's just me, but for example I prefer no indentation, no bold title and an underlined title... to check with SBU). In any case:
- this should be implemented like `we-row`. I would name this new feature `we-group`. The title is automatically generated from the string attribute, like `we-row`.
- cannot use pl-1 mt-0 as their spacing depends on the theme, you should add css to the `we-group` element